Cakes-Co/css/style.css

```diff
@@ -0,0 +1,312 @@
+/*  Add your styling here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --pink: rgb(242 200 217);
+  --darker-pink: rgb(201, 40, 105);
+  --light-pink: rgb(249 237 244);
+  --background-color: var(--white);
+  --gold: rgb(160 150 111);
+  --white: rgb(255, 255, 255);
```

Comment:
I like how you've made a CSS var with an abstract name background-color, this way it still makes sense if you decide to change the color inside the var, I might extend it to  the link colors and other colors e.g. --link-color